server.ts

```diff
@@ -1,5 +1,5 @@
 import 'source-map-support/register';
-import env from './env';
+const env = require('./env');
```

Comment:
Please avoid mixing `require`s and ES module `import`s. Suggest to use `import` everywhere for consistency with the front end codebases.